samples/basic/fade_led/nrf52_pca10040.overlay

```diff
@@ -0,0 +1,17 @@
+&pwm0 {
+	status = "ok";
+	ch0-pin = <17>;
```

Comment:
This file is not needed. The `pwm0` node is configured properly at the board level.

Reply:
Part of demonstration on how to use SW-PWM driver instead of HW.
Removed.